scenes/game_elements/props/decoration/decoration.gd

```diff
@@ -0,0 +1,17 @@
+# SPDX-FileCopyrightText: The Threadbare Authors
+# SPDX-License-Identifier: MPL-2.0
+@tool
+class_name Decoration
+extends Node2D
+
+
+func _set(property: StringName, value: Variant) -> bool:
+	return PropertyUtils.set_child_property(self, property, value)
+
+
+func _get(property: StringName) -> Variant:
+	return PropertyUtils.get_child_property(self, property)
```

Comment:
Something that wasn't immediately obvious to me, but which now I've read the documentation I see works rather neatly, is that if `_set` returns `false` or `_get` returns `null`, then Godot will fall back to normal property lookup.